Bottom: 704df4239d203b287a45f6ddb38ee56d7f4a4307
Top:    21f6fd5398dd43dd7c79973a06007d8a10ed2e8a
Author: Ian Kent <raven@themaw.net>
Date:   2013-03-21 12:17:11 +0800

Refresh of autofs-5.0.5-add-master-read-wait-option.patch

---

diff --git a/daemon/automount.c b/daemon/automount.c
index 05196f4..2b2e57f 100644
--- a/daemon/automount.c
+++ b/daemon/automount.c
@@ -1943,7 +1943,7 @@ int main(int argc, char *argv[])
 	int logpri = -1;
 	unsigned ghost, logging, daemon_check;
 	unsigned dumpmaps, foreground, have_global_options;
-	unsigned master_read, master_wait;
+	unsigned master_read;
 	int master_wait;
 	time_t timeout;
 	time_t age = time(NULL);
@@ -2312,7 +2312,7 @@ int main(int argc, char *argv[])
 		 * a signal is received, in which case exit returning an
 		 * error.
 		 */
-		if (!do_master_read_master(master_list, age, -1)) {
+		if (!do_master_read_master(master_list, age, master_wait)) {
 			logerr("%s: failed to read master map!",
 			       program);
 			master_kill(master_list);
